cmd/docker/docker.go

```diff
@@ -57,6 +57,7 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags = cmd.Flags()
 	flags.BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
 	flags.StringVar(&opts.ConfigDir, "config", cliconfig.Dir(), "Location of client config files")
+	flags.BoolVarP(&opts.TLSGetPass, "tlsgetpass", "", cliconfig.TLSGetPass(), "Request passphrase if needed to decrypt TLS keys")
```

Comment:
I think it's probably fine to just prompt for a passphrase if it was encrypted - it just returned an error before, and so long as we message that the TLS key is encrypted, and that's why they're being asked for a passphrase now, that seems to be a fine behavior change.  (we probably just need to make sure the user can control-C or something out of the password prompt if they wanted).